roofit/roofitcore/src/RooAbsMinimizerFcn.cxx

```diff
@@ -0,0 +1,526 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitCore                                                       *
+ * @(#)root/roofitcore:$Id$
+ * Authors:                                                                  *
+ *   AL, Alfio Lazzaro,   INFN Milan,        alfio.lazzaro@mi.infn.it        *
+ *   PB, Patrick Bos, Netherlands eScience Center, p.bos@esciencecenter.nl   *
+ *                                                                           *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+#ifndef __ROOFIT_NOROOMINIMIZER
```

Comment:
Is this `ifndef` really needed? When would we build with `__ROOFIT_NOROOMINIMIZER`?

Reply:
It was already there, but seems unused. Wouter confirms that it is obsolete, so could be removed in all places that it is still used. Separate PR probably?

Reply:
Ah sorry, I didn't realize it was there before. I agree, everything that is cleaning up old stuff is a task for a separate PR.

Reply:
No, not needed. Agree about a separate PR, where things like [this part here](https://github.com/root-project/root/blob/7c4f9321a3fcbda598da6b815409a9a5ed72a53e/roofit/roofitcore/src/RooAbsPdf.cxx#L1599-L1783) could be condensed down into the one branch that's actually used.

Reply:
I will just remove them when I encounter them.

Reply:
Oh weird, I didn't see your replies when I typed that I would just remove them, should have refreshed the page. I didn't actually notice the part that you linked @hageboeck. The ifdef's I encountered so far were completely inconsequential (those that switch on/off the entire file, which I think really is useless), so perhaps those can be safely removed while the one you mention would need a little extra motivation and double checking.

Reply:
Yes, please do remove them here, and Jonas can take care of the ones that are left in roofitcore.
#8604